### ISSUE TITLE
The extension seems to ignore the executionEnvironment configuration

### ISSUE THREAD

**Issue by Xevib** [ansible/vscode-ansible#314]
### Summary

# TL;DR
With the configuration `ansible.executionEnvironment.enabled=false` the command "Run playbook via ansible-navigator run" still runs the ansible-navigator on the enviroment(docker)

# Description
- I'm trying to execute a playbook that contains a collection that it's not installed on the docker image. If execute on vscode I get a message saying that the collection is not installed. I tried to dissable the environment but seems that it stills working.
- I expected that the beheivour wold be like if i execute ` ansible-navigator run '/deploy.yml' --execution-environment false` and ansible-navigator use the collections installed on my local ansible

### Extension version

0.6.

### VS Code version

1.6.2

### Ansible Version

```shell
ansible [core 2.11.6] 
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/xevi/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/xevi/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.8.10 (default, Sep 28 2021, 16:10:42) [GCC 9.3.0]
  jinja version = 2.10.1
  libyaml = True
```

### OS / Environment

OS  Ubuntu 20.04.3 LTS 
ansible-lint ansible-lint 5.2.1 using ansible 2.11.6
ansible-navigator ansible-navigator 1.1.0

**Comment by ssbarnea** [comment]
We can implement something like:
```
Navigator extra arguments: default empty.
When empty we will add --ee,... when calling navigator but when you configure it, we will only call navigator with your custom options.
```

**Comment by ganeshrn** [comment]
We do plan to add client option to set the navigator config file which if set will take precedence over client EE configuration options and the config file path value will be passed to `ansible-navigator run` command line .


```
ansible.ansibleNavigator.configFilePath: <path to config file>.    # takes precedence over EE client options explicitly set
```

If the `ansible-navigator` config path is not set in the client options in that case client will read the below EE options and pass as command line option to `ansible-navigator run`
```
    "ansible.executionEnvironment.enabled": false,
    "ansible.executionEnvironment.containerEngine": "auto",
    "ansible.executionEnvironment.image": "quay.io/ansible/ansible-navigator-demo-ee:0.6.0",
    "ansible.executionEnvironment.pullPolicy": "missing",
```

**Comment by Xevib** [comment]
Thank you @ssbarnea & @ganeshrn .
If I can help you to test somthing just contactme
